Mathlib/Order/Antisymmetrization.lean

```diff
@@ -27,39 +28,56 @@ such that `a ≤ b` and `b ≤ a`.
 
 open Function OrderDual
 
-variable {α β : Type*}
+variable {α β : Type*} {a b c d : α}
 
 section Relation
 
 variable (r : α → α → Prop)
 
-/-- The antisymmetrization relation. -/
+/-- The antisymmetrization relation `AntisymmRel r a b` means that both `r a b` and `r b a`. -/
```

Comment:
Grammar seems off

Reply:
How exactly? Not a native speaker but as someone who's spoken English most my life this doesn't ring any alarm bells.

Reply:
Firstly, `AntisymmRel r` is a relation but `AntisymmRel r a b` is a proposition. Secondly, I would say "`r a b` and `r b a` both **holds**". Thirdly (not your fault), antisymmetrization is an operation/function, and `AntisymmRel r` should probably be called the antisymmetriz**ed** relation rather than the antisymmetrization relation.

I suggest that you keep it simple:
```suggestion
/-- The antisymmetrized relation `AntisymmRel r` is defined so that `AntisymmRel r a b ↔ r a b ∧ r b a`. -/
```
If you want to describe it with words, I'd say "`AntisymmRel r` is the conjunction of `r` with its flip".

Reply:
Sorry, scratch the third part. `AntisymmRel r` is not the antisymmetrized relation since it's not antisymmetric. It's the equivalence relation to take quotient by to get the Antisymmetrization, so "antisymmetrization relation" is a proper / correct name.